assets/css/system-info.css

```diff
@@ -0,0 +1,11 @@
+#system-info-textarea {
```

Comment:
@Gallmond @CWDN would like to get your thoughts on this approach. A dedicated CSS and JS file when required.

Idea being that it's easy to find the related resources seeing as my IDE does not link anything! (That might just be my IDE not being setup for Wordpress more than anything)

I'm not married to the idea so happy to go towards a general `custom.css` files a `custom.js` or something like that if people prefer!

Reply:
I could be wrong but I think we tend to add to the `sass` within `assets/css/sass/gathercontent-importer.scss` then it will get compiled and pulled in automatically. Saves enqueuing more files.

Think you can do this with the JS as well and get grunt to compile the JS for you, it will at least be minified and all the other stuff required for Wordpress.